shell/platform/darwin/common/framework/Source/FlutterChannels.mm

```diff
@@ -128,26 +120,19 @@ - (void)resizeChannelBuffer:(NSInteger)newSize {
 ////////////////////////////////////////////////////////////////////////////////
 @implementation FlutterError
 + (instancetype)errorWithCode:(NSString*)code message:(NSString*)message details:(id)details {
-  return [[[FlutterError alloc] initWithCode:code message:message details:details] autorelease];
+  return [[FlutterError alloc] initWithCode:code message:message details:details];
 }
 
 - (instancetype)initWithCode:(NSString*)code message:(NSString*)message details:(id)details {
   NSAssert(code, @"Code cannot be nil");
   self = [super init];
   NSAssert(self, @"Super init cannot be nil");
-  _code = [code retain];
-  _message = [message retain];
-  _details = [details retain];
+  _code = code;
+  _message = message;
+  _details = details;
```

Comment:
And these properties are `copy` but you only get that on the property setter, not when you set the synthesized ivar.
```suggestion
  _code = [code copy];
  _message = [message copy];
  _details = [details copy];
```